### PR TITLE
chore: added AsaKatsu in OpenSource Projects

### LINKED ISSUE #547: chore: added AsaKatsu in OpenSource Projects
### Site URL

https://github.com/asakatsuOrg/AsaKatsuProject

### Description of the above provided site

A project where you can keep track of your goal's progress, and contribute to open source in the same time.

AsaKatsu is a Japanese term for waking up a little early to do something productive, in line with a goal. For example; read 4 books per month, do 100 days challenge, etc.

### Category

Frontend

### Sub Category

Cat - OpenSource  ,   SubCat - Projects 

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by parma1111**
## Fixes Issue

Closes #547 

## Changes proposed

A project where you can keep track of your goal's progress, and contribute to open source in the same time.

## Screenshots

Screenshot (806)

## Note to reviewers

Any kind of feedback would be appreciated.

### CODE REVIEW

**Review by rupali-codes — APPROVED** [review]
Thank you Parma, looks good to me

**Review by ujjawaltyagii — APPROVED** [review]
Lgtm 👍

**Comment by parma1111** [comment]
@CBID2

File: database/opensource/projects.json [hunk 1]
```diff
@@ -12,5 +12,12 @@
     "url": "https://github.com/IAmTamal/Milan",
     "category": "opensource",
     "subcategory": "projects"
+  },
+  {
+    "name": "AsaKatsu",
+    "description": "A project where you can keep track of your goals' progress. 'AsaKatsu' is a Japanese term for waking up a little early to do something productive, aligned with a specific goal.",
```
Comment (CBID2) [inline database/opensource/projects.json:18]:
```suggestion
    "description": 'AsaKatsu' is a Japanese term for waking up a little early to do something productive, aligned with a specific goal. With this philosophy in mind, this product helps you keep track of your goals' progress.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Looks pretty good so far @parma1111! :) I just made a suggestion on how to improve your description.

**Comment by parma1111** [comment]
> Looks pretty good so far @parma1111! :) I just made a suggestion on how to improve your description.

> Hey @CBID2! :) I have changed the description, now you can review the PR.

**Review by CBID2 — APPROVED** [review]
LGTM! 😊

**Comment by CBID2** [comment]
@rupali-codes, for some reason, I can’t merge it

**Comment by rupali-codes** [comment]
> @rupali-codes, for some reason, I can’t merge it

You aren't seeing the option again?

**Comment by rupali-codes** [comment]
@CBID2 could you try now?

**Comment by parma1111** [comment]
@rupali-codes @CBID2

**Comment by CBID2** [comment]
> @rupali-codes @CBID2

Still can't merge it

**Comment by CBID2** [comment]
> > > @rupali-codes @CBID2
> > 
> > 
> > Still can't merge it. Apparently, it is missing successful Preview and Production deployments @rupali-codes .
> 
> it has deployed successfully

I agree with you @rupali-codes, but the issue remains(see below screenshot)
screenshot of issue

**Comment by parma1111** [comment]
Now, what should I change in this PR - @rupali-codes @CBID2

**Comment by rupali-codes** [comment]
@CBID2 I'm merging this PR by bypassing the rules and let's look for a solution

**Pull request merged**